deployment/.env

```
@@ -0,0 +1,20 @@
+####### DATABUS SETTINGS #######
+DATA_PATH="../../databus-data"
+# RESOURCE BASE URL - This should match the DNS entry pointing to your server
+DATABUS_RESOURCE_BASE_URL=http://localhost:3000
+# do not use qoutation marks for the value here and no trailing space
+DATABUS_NAME=My Local Databus
+DATABUS_ABSTRACT=A local instance for development
+DATABUS_ORG_ICON=
+DATABUS_BANNER_COLOR= # Default is #81b8b2
+DATABUS_TITLE_COLOR= # Default is white
+# OIDC SETTINGS - Only use these values for testing and enter your own OIDC provider values
+DATABUS_OIDC_ISSUER_BASE_URL=https://kilt.eu.auth0.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABUS_OIDC_ISSUER_BASE_URL key should go before the DATABUS_ORG_ICON key

(UnorderedKey)